src/Confluent.Kafka/AdminClient.cs

```diff
@@ -207,6 +207,23 @@ private List<DeleteAclsReport> extractDeleteAclsReports(IntPtr resultPtr)
             }).ToList();
         }
 
+        private List<DeleteConsumerGroupOffsetsReport> extractDeleteConsumerGroupOffsetsReports(IntPtr resultPtr)
+        {
+            IntPtr groupsOffsetsResultPtr = Librdkafka.DeleteConsumerGroupOffsets_result_groups(resultPtr, out UIntPtr resultCountPtr);
```

Comment:
`resultCountPtr` -> `resultCount`. Although the type is `UIntPtr`, it is semantically not a pointer. The type merely indicates the size of the type (which is platform dependent).

Reply:
Updated

Reply:
no it's not ...

Reply:
Sorry missed in the last commit, fixed now.